src/libxively/sft/xi_sft_logic.c

```diff
@@ -293,10 +390,7 @@ xi_sft_on_message( xi_sft_context_t* context, xi_control_message_t* sft_message_
 
                     /* continue the update package download  */
                     {
-                        context->update_current_file =
-                            xi_control_message_file_update_available_get_next_file_desc_ext(
```

Comment:
please also remove the header and implementation of this old function.

Reply:
:+1: